pipeline.py

```diff
@@ -96,19 +102,91 @@ def sha(s):
             open(pickle_name, 'wb'),
             protocol=cPickle.HIGHEST_PROTOCOL)
 
+    # -- plot distributions:
```

Comment:
Everything from line 105 to line 157 has been removed this script and substituted with the few lines of code right above. You can go ahead and delete it.